### DISCUSSION TITLE
How to install Budibase to existing droplet?

### DISCUSSION THREAD

**Discussion by burner918** [Budibase/budibase#5332] [Help]
Hi,

I'm new to DigitalOcean and already had a droplet setup with an existing app in one of my folders. The thing is I wish to install Budibase to the same droplet, i.e. I don't wish to create a new droplet to install Budibase. I can't seem to figure out how to do this and the documentation that I've come across only shows me the one-click installation of Budibase from the marketplace in a new droplet. Any help would be much appreciated. Thanks in advance.

**Comment by shogunpurple** [comment]
Hi @burner918 -

So since you have a droplet already, you just install budibase as you would on any other linux machine (assuming your droplet is linux)

https://docs.budibase.com/docs/docker-compose
